service/DeliveryServiceModel.go

```diff
@@ -0,0 +1,25 @@
+package service
+
+import (
+	"fmt"
+	"github.com/Baraulia/COURIER_SERVICE/dao"
+	"log"
+)
+
+type DelServService struct {
```

Comment:
you have `DelServ` here, but `NewDelivServ` below. Though I would propose not to abbreviate it so much, at least the word delivery